markdown/ch3.markdown

```diff
@@ -1,5 +1,15 @@
 # Networks of Image Regions with ndimage
 
+> "Tyger Tyger, burning bright,
```

Comment:
You can do newlines in Markdown without starting a new paragraph: add two spaces at the end. Though there might have been an issue with that in the HTML conversion — @stefanv?